packages/@react-spectrum/s2/src/ComboBox.tsx

```diff
@@ -148,6 +149,114 @@ const iconStyles = style({
   }
 });
 
+export let listbox = style<{size: 'S' | 'M' | 'L' | 'XL'}>({
+  width: 'full',
+  boxSizing: 'border-box',
+  maxHeight: '[inherit]',
+  // TODO: Might help with horizontal scrolling happening on Windows, will need to check somehow. Otherwise, revert back to overflow: auto
+  overflowY: 'auto',
+  overflowX: 'hidden',
+  fontFamily: 'sans',
+  fontSize: 'control'
+});
+
+export let listboxItem = style({
+  ...focusRing(),
+  boxSizing: 'border-box',
+  borderRadius: 'control',
+  font: 'control',
+  '--labelPadding': {
+    type: 'paddingTop',
+    value: centerPadding()
+  },
+  paddingBottom: '--labelPadding',
+  backgroundColor: { // TODO: revisit color when I have access to dev mode again
```

Comment:
need help checking this?

Reply:
oh yeah that's probably from copy paste. it's been there prior to starting this PR but i can look into it

Reply:
took a look at the figma and it seems correct. gonna remove the comment